crates/node/migrations/20231128120351_transactions-table.sql

```diff
@@ -12,6 +12,7 @@ DROP TYPE IF EXISTS transaction_kind;
 CREATE TYPE transaction_kind AS ENUM ('empty','transfer', 'stake', 'unstake', 'deploy', 'run', 'proof', 'proofkey', 'verification', 'cancel');
 
 CREATE TABLE transaction (
+    author VARCHAR(130) NOT NULL,
```

Comment:
This is a bit drive-by change and I wouldn't normally combine these, but gotta move fast :see_no_evil: